sdk/nodejs/src/models/populatedPBUser.ts

```diff
@@ -1,84 +1,50 @@
-import { DVCCustomDataJSON } from '../types'
-import * as packageJson from '../../package.json'
-import { DevCycleUser } from './user'
-import os from 'os'
 import { ProtobufTypes } from '@devcycle/bucketing-assembly-script'
+import {
+    DVCPopulatedUser,
+    DevCycleUser,
+    DVCCustomDataJSON,
+} from '@devcycle/js-cloud-server-sdk'
+import { getNodeJSPlatformDetails } from '../utils/platformDetails'
 
-export class DVCPopulatedUser implements DevCycleUser {
-    user_id: string
-    email?: string
-    name?: string
-    language?: string
-    country?: string
-    appVersion?: string
-    appBuild?: number
-    customData?: DVCCustomDataJSON
-    privateCustomData?: DVCCustomDataJSON
-    readonly lastSeenDate: Date
-    readonly createdDate: Date
-    readonly platform: string
-    readonly platformVersion: string
-    readonly sdkType: 'server'
-    readonly sdkVersion: string
-    readonly hostname: string
-
+export class DVCPopulatedPBUser extends DVCPopulatedUser {
```

Comment:
does this really need to be its own specially named class? it kinda bleeds details like "protobuf" to other parts of the SDK that don't really care, and just wanted a PopulatedUser

Maybe you could just have a helper method like `populatedUserToProtobuf` that gets called in the one place it needs to do that conversion?

Reply:
I'll take a look, I wonder if you can do transparent extensions like you can in other languages.